atlasdb-commons/src/test/java/com/palantir/util/SafeShutdownRunnerTest.java

```diff
@@ -79,9 +79,8 @@ public void slowTasksTimeOutWithoutThrowing() {
         runner.shutdownSafely(blockingUninterruptibleRunnable);
         runner.shutdownSafely(blockingUninterruptibleRunnable);
 
-        verify(blockingUninterruptibleRunnable, times(2)).run();
-
         closeAndAssertNumberOfTimeouts(runner, 2);
+        verify(blockingUninterruptibleRunnable, times(2)).run();
```

Comment:
This also flaked. Re-ordering should enforce that the shutdown runner has closed and thus has definitely run both tasks.